scenario/build/generate.xml

```diff
@@ -13,7 +13,6 @@
                 <item name="300" xsi:type="array">
                     <item name="config-file-exists" xsi:type="object" priority="100">Magento\MagentoCloud\Config\Validator\Build\ConfigFileExists</item>
                     <item name="deprecated-ini" xsi:type="object" priority="200">Magento\MagentoCloud\Config\Validator\Build\UnsupportedBuildOptionsIni</item>
-                    <item name="stage-config-deprecated-variables" xsi:type="object" priority="300">Magento\MagentoCloud\Config\Validator\Build\StageConfigDeprecatedVariables</item>
```

Comment:
Please adjust the priority for other steps.

Reply:
Done